### ISSUE TITLE
Wiremod

### ISSUE THREAD

**Issue by MSReporter** [Metastruct/issues#196]
repeater repeating replacing 

Bug reported by '<color=36,0,130>holz' (STEAM_0:1:5897).

**Comment by PotcFdk** [comment]
I... I don't follow you.

**Comment by TW1STaL1CKY** [comment]
He showed me this and I told him to report it, but he hasn't put much at all... I took a few screenshots before it was removed, but I wasn't able to look into detail how it was wired.

Here is what he had:
![Setup](https://dl.dropboxusercontent.com/u/204143210/ShareXScn/2016-09/2016-09-18_16-41-15.png)
If I remember correctly, it was 2 Repeat Wire Gates, a Replace Wite Gate, a constant string Wire Value and a Wire Button (the clock).

![The value](https://dl.dropboxusercontent.com/u/204143210/ShareXScn/2016-09/2016-09-18_16-50-01.png)
The constant value held "(.*)*" and was wired to the Replace Wire Gate.

![The Replace gate](https://dl.dropboxusercontent.com/u/204143210/ShareXScn/2016-09/2016-09-18_16-49-43.png)
The Replace gate was wired to both Repeat gates.

![The Repeat gate](https://dl.dropboxusercontent.com/u/204143210/ShareXScn/2016-09/2016-09-18_16-48-45.png)
The 2 gates originally only had "(.*)*" set on them but changed to that after pressing the button once. It can't really be seen from the picture, but I think the 2 Repeat gates were wired together. They were also connected to the button. 

Everytime the button is pressed, the server would freeze for 3-10 seconds, then would print this in the server console:
`[774] Wire Gate: not enough memory`
